modules/web-console/frontend/public/stylesheets/style.scss

```diff
@@ -269,7 +269,9 @@ body {
 
     min-height: 100vh;
     display: grid;
-    grid-template-columns: auto 1fr;
+    // Zero min width prevents from content blowing out column width
+    // https://stackoverflow.com/a/43312314/333777
+    grid-template-columns: auto minmax(0, 1fr);
```

Comment:
If long text is on the left side, shouldn't it be `minmax(0, auto) 1fr`?

Reply:
@vikvorona left column if for navigation, `1fr` in right column causes the issue.